### PR TITLE
feat(Security): added SOC2 doc acquisition information

### PR DESCRIPTION
**Pull request by jeff-colucci**

### CODE REVIEW

**Review by ally-sassman — DISMISSED** [review]

**Review by ally-sassman — DISMISSED** [review]

File: src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/iso.mdx [hunk 1]
```diff
@@ -60,6 +60,10 @@ The following applies to the New Relic platform:
   </tbody>
 </table>
 
+<Callout variant="important">
+If you require a copy of New Relic's full ISO 27001 certification letter, reach out to your New Relic account executive.
```
Comment (paperclypse) [inline src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/iso.mdx:64]: Shouldn't it be "account manager" or "account representative"?

Comment (paperclypse) [reply src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/iso.mdx:64]: I'm seeing "account representative" used most often in our docs

**Review by paperclypse — DISMISSED** [review]
Approved, once you fix this one minor wording thing, @jeff-colucci

File: src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/iso.mdx (hunk 1 above)
Comment (jeff-colucci) [reply src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/iso.mdx:64]: That's what I thought as well! I co-wrote this with someone in legal and apparently account executive is the correct term to use.

**Comment by jeff-colucci** [comment]
Merging with prior approval

**Pull request merged**